yamale/validators/validators.py

```diff
@@ -21,7 +12,236 @@ class String(Validator):
     constraints = [con.LengthMin, con.LengthMax, con.CharacterExclude]
 
     def _is_valid(self, value):
-        return isstr(value)
+        return util.isstr(value)
+
+
+class Pathname(String):
```

Comment:
It seems like inheriting from `String` doesn't give `Pathname` much here. It's easy enough just tor reuse `isstr`. Inheriting also means you bring `String`s `constraints` along which doesn't really apply to `Pathname`.